presidio_evaluator/evaluation/evaluator.py

```diff
@@ -258,7 +318,9 @@ def calculate_score(
         entity_precision = {}
         n = {}
         if not entities:
-            entities = list(set([x[0] for x in all_results.keys() if x[0] != "O"]))
+            entities1 = list(set([x[0] for x in all_results.keys() if x[0] != "O"]))
+            entities2 = list(set([x[1] for x in all_results.keys() if x[1] != "O"]))
+            entities = list(set(entities1).union(set(entities2)))
```

Comment:
update `entities` to everything in both the predicted and actual lists of entities, instead of just actual.